README.md

```diff
@@ -1,30 +1,17 @@
-# HPE OneView golang bindings
-
-[![Build Status](https://travis-ci.org/HewlettPackard/oneview-golang.svg?branch=master)](https://travis-ci.org/HewlettPackard/oneview-golang)
```

Comment:
Oops, I was just trying to fix the signature without making drastic changes. I pulled this out of my readme so I wouldn't be getting builds passing or failing from your repo onto mine. I'll fix this.